src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/disk_encryption.py

```diff
@@ -180,7 +193,10 @@ def encrypt_vm(resource_group_name, vm_name,  # pylint: disable=too-many-locals,
                        "the encryption will finish shortly")
 
 
-def decrypt_vm(resource_group_name, vm_name, volume_type=None, force=False):
+def decrypt_vm(resource_group_name,
```

Comment:
NIT: are you using any python editor to format this line? We have updated pylint configuration to allow bigger line width, so you should back out